src/main/java/callofcactus/map/MapFiles.java

```diff
@@ -0,0 +1,26 @@
+package callofcactus.map;
+
+/**
+ * Created by Teun on 21-11-2015.
+ */
+public class MapFiles {
+
+    public static final String COMPLICATEDMAP = "complicatedmap.tmx";
+
+    public enum MAPS {
+        COMPLICATEDMAP
+    }
+
+    public MAPS getMap(String mapname) {
+        return MapFiles.MAPS.valueOf(mapname);
+    }
+
+    public static String getFileName(MAPS map) {
+        switch (map) {
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Switch statements should have a default label](https://www.codacy.com/app/guushamm/Call-of-Cactus/file/1385244492/issues/source?bid=2979575&fileBranchId=2980149#l19)